src/screens/CommentModal.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+
+import { StatusBar } from 'expo-status-bar';
+import { Platform, StyleSheet, View, Button } from 'react-native';
+import { Title, TextInput } from 'react-native-paper';
+
+import EditScreenInfo from 'components/EditScreenInfo';
+import ViewContainer from 'src/components/ViewContainer';
+
+const styles = StyleSheet.create({
+  input: {
+    width: '90%',
+  },
+  separator: {
+    marginVertical: 30,
+    height: 1,
+    width: '80%',
+    backgroundColor: '#eee',
+  },
```

Comment:
what's the difference between this and the base modal screen? can you reuse the latter?